djbetty/storage.py

```diff
@@ -99,3 +99,23 @@ def url(self, name, ratio="original", width=600, format="jpg", fixed=0):
             ratio=ratio,
             width=width,
             format=format)
+
+    def animated_url(self, name, format="gif", fixed=0):
+        id_string = ""
+        for index, char in enumerate(str(name)):
+            if index % 4 == 0 and index != 0:
+                id_string += "/"
+            id_string += char
+
+        # Allows request to use fixed url from settings
+        base_url = getattr(settings, 'BETTY_FIXED_URL', None) if fixed else None
+        if not base_url:
+            base_url = self.base_url
+
+        base_url = base_url.rstrip('/')
```

Comment:
First part of this function shares identical code with `def url`, I would move into a common helper function (esp tricky logic to build id_string)